### PR TITLE
Adjust visuals of osu!mania barlines to be less present

### LINKED ISSUE #19611: osu!mania barlines are too present (and no classic skin support yet)
### Discussed in https://github.com/ppy/osu/discussions/19608

<div type='discussions-op-text'>

<sup>Originally posted by **coiledmocha** August  7, 2022</sup>
What are these weird lines and how do I turn them off. Very distracting, and doens't seem normal. Thanks in advance.

osumaniaweirdlines
</div>

### PR DESCRIPTION
**Pull request by peppy**
Roughly matches the new design. Metrics adjusted to fit with the existing design.

Closes #19611 maybe?

Before:

osu Game Rulesets Mania Tests 2022-08-08 at 08 41 05

After:

osu Game Rulesets Mania Tests 2022-08-08 at 08 40 49

### CODE REVIEW

**Comment by bdach** [comment]
Code looks fine and the bar lines do look less intrusive, but there's one side thing that's spurred by this change which is bothering me slightly. When notes are present on the screen alongside the bar lines, the end result looks ever so slightly... off?

osu_2022-08-08_21-23-42

Note that the major bar line indicators to the side of the stage are vertically centred *to* the bar line, while the note that I'm supposed to be hitting at that bar line is... *above* the line? I'm not a VSRG person so maybe I just don't know better, am I wrong for being bothered by this?

Not saying it's something that needs to be addressed here, just giving a subjective opinion I guess.

File: osu.Game.Rulesets.Mania/Objects/Drawables/DrawableBarLine.cs [hunk 1]
```diff
@@ -16,56 +13,45 @@ namespace osu.Game.Rulesets.Mania.Objects.Drawables
...
                 RelativeSizeAxes = Axes.Both,
-                Colour = new Color4(255, 204, 33, 255),
+                Alpha = barLine.Major ? 0.5f : 0.2f
             });
 
             if (barLine.Major)
             {
-                AddInternal(new EquilateralTriangle
+                Vector2 size = new Vector2(22, 6);
+                const float triangle_offset = 4;
```
Comment (bdach) [inline osu.Game.Rulesets.Mania/Objects/Drawables/DrawableBarLine.cs:34]: Also a nitpick, but probably worth renaming this to `line_offset` or something. No triangles in sight anymore here.

**Comment by mk56-spn** [comment]
> Code looks fine and the bar lines do look less intrusive, but there's one side thing that's spurred by this change which is bothering me slightly. When notes are present on the screen alongside the bar lines, the end result looks ever so slightly... off?
> 
> osu_2022-08-08_21-23-42
> 
> Note that the major bar line indicators to the side of the stage are vertically centred _to_ the bar line, while the note that I'm supposed to be hitting at that bar line is... _above_ the line? I'm not a VSRG person so maybe I just don't know better, am I wrong in being bothered by this?
> 
> Not saying it's something that needs to be addressed here, just giving a subjective opinion I guess.

 standard for vsrg skins
download
this popular custom mania skin  is not vertically centered im p sure

**Comment by peppy** [comment]
Yeah, that's generally how bar lines work in musical scores and I think most people are happy with them displaying at the pixel of timing change, rather than centered on hitobjects.

**Review by smoogipoo — APPROVED** [review]

**Comment by win50117** [comment]
How to disable line symbol? (origin is triangle)
reason why I haven't moved to lazer yet. 
I hate this between barline. It that me distracting and always click wrong key.

**Comment by peppy** [comment]
This will be customisable in the near future.

**Pull request merged**